content/blog/2018-02-25-using-parse-in-nativescript-vue.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Using Parse in Nativescript-vue
+authors: [damain]
+toc: true
+---
+
+### Using Parse in Nativescript-vue
```

Comment:
There is no need to add this title, as the above title (from the front matter) will be displayed on the page.